doc/developer/01_getting_started.md

```diff
@@ -0,0 +1,70 @@
+![hitobito logo](https://user-images.githubusercontent.com/9592347/184715060-351453d4-d066-4ff6-8f82-95d3b524b62f.svg)
+## Getting started 🍀🍀🌅
+
+To get started with the project following steps are required:
+
+### Stage 1: Get Hitobito to run locally
+
+- [ ] Setup Project locally, please follow the instructions at [Development](https://github.com/hitobito/development/) 
+for the recommended 🚢 docker development setup.
+
+
+- [ ] Make sure Tests 👨‍🔬 are running, look at [Tests](02_testing)
+
+
+- [ ] Check if you can access the mailcatcher which run on [http://localhost:1080](http://localhost:1080)
+
+
+- [ ] Look 👁️ at the table with all specific rake tasks in it, try some of them if interested [Rake Tasks](#specific-rake-tasks) 
+
+Run the following rake tasks inside rails, rails-test container:
+
+| Task                      | Beschreibung                                                                                           |
```

Comment:
can you extract this table to a separate file?
does it make sense to ask a new developer to just randomly check those commands?